de-lib/src/main/java/org/iplantc/de/client/models/apps/App.java

```diff
@@ -1,23 +1,20 @@
 package org.iplantc.de.client.models.apps;
 
+import com.google.gwt.user.client.ui.HasName;
```

Comment:
Totally trivial, but we have some [code style settings](https://github.com/cyverse-de/ui/tree/master/iplant/code-style) that make sure the iplant/cyverse imports come before anything else.